recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/files/pvr-addons/etc/udev/rules.d/72-pvr-seat.rules

```diff
@@ -0,0 +1,7 @@
+############################################################################ ###
+# File 72-pvr-services.rules
+# Copyright Copyright (C) 2013-2016 Renesas Electronics Corporation
+# License Strictly Confidential.
+#### ###########################################################################
+# Rules for PowerVR device nodes
+SUBSYSTEM=="drm", DEVPATH=="/devices/platform/soc/fd000000.gsx/drm/*", TAG+="seat", TAG+="master-of-seat", ENV{ID_AUTOSEAT}="1", ENV{ID_SEAT}="seat1"
```

Comment:
Will this work on all Renesas platforms (and under virtualization?)

Reply:
it does at the moment if run as Dom0. After we move AGL to real DomD we will have to change this. ATM it is not clear what will be the right path...

Reply:
moved to the version supplied during the build/install by the PVR UM itself